modules/acquia_cms_search/acquia_cms_search.module

```diff
@@ -161,3 +162,80 @@ function acquia_cms_search_search_api_server_update(EntityInterface $entity) {
 function acquia_cms_search_form_acquia_cms_solr_search_form_alter(array &$form) {
   $form['#submit'][] = AcquiaSearchFacade::class . '::submitSettingsForm';
 }
+
+/**
+ * Implements hook_modules_installed().
+ */
+function acquia_cms_search_modules_installed(array $modules) {
+  $module_handler = Drupal::moduleHandler();
+  // Category facet is created if content type modules are installed.
+  if ($module_handler->moduleExists('acquia_cms_person') ||
+  $module_handler->moduleExists('acquia_cms_place') ||
+  $module_handler->moduleExists('acquia_cms_article') ||
+  $module_handler->moduleExists('acquia_cms_event') ||
```

Comment:
@panshulK I think we need to add PHPUnit tests as well for all this to make sure when this modules is/are enabled, facet gets created automatically successfully.

Reply:
Done

Reply:
Done